avocado/utils/nvme.py

```diff
@@ -0,0 +1,82 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# This code was inspired in the autotest project,
+#
+# client/base_utils.py
+#
+# Copyright: 2022 IBM
+# Authors : Naresh Bannoth <nbannoth@linux.vnet.ibm.com>
+
+
+"""
+Nvme utilities
+"""
+
+
+import os
+from avocado.utils import process
+
+
+class NvmeError(Exception):
+    """
+    Generic DiskError
```

Comment:
please reword this as we are catching here Nvme error, just a suggestion can we narrow down Exception as it has broader scope